lib/builder/structs/tx_build.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Stellar.Builder.Structs.TxBuild do
+  @moduledoc """
+  `TxBuild` struct definition.
+  """
+  alias Stellar.Builder.Structs.{Memo, TimeBounds, Transaction, TxEnvelope}
+
+  @type t :: %__MODULE__{tx: Transaction.t(), signatures: list(), tx_envelope: TxEnvelope.t()}
```

Comment:
Is there a way to be more specific about the type(s) in the signatures list?

Reply:
@luishurtado There is no type because this is not implemented yet, #22 will implement it.